1/contrib/s2i/run

```diff
@@ -69,6 +60,12 @@ fi
 
 rm -rf /tmp/war
 
+echo "Processing Jenkins Kubernetes configuration (${CONFIG_PATH}) ..."
+envsubst < "${CONFIG_PATH}.tpl" > "${CONFIG_PATH}" && rm -f "${CONFIG_PATH}.tpl"
```

Comment:
need to balance this against the ability for users to supply their own configuration file that should not be overwritten by this logic.  maybe only do this if a config.xml doesn't exist?  except i'm assuming you want to do this always so you can pick up env changes.